ERCS/erc-4337-agg.md

```diff
@@ -0,0 +1,145 @@
+---
+eip:
+description: An ERC-4337 improvement to aggregation of all UserOperation signatures in a bundle
+title: Signature Aggregation for Account Abstraction
+author: Vitalik Buterin (@vbuterin), Yoav Weiss (@yoavw), Dror Tirosh (@drortirosh), Shahaf Nacson (@shahafn), Alex Forshtat (@forshtat)
+discussions-to:
```

Comment:
Please create a discussions topic in Eth Magicians

Reply:
Created.